SignalShareExtension/ShareViewController.swift

```diff
@@ -75,6 +68,22 @@ public class ShareViewController: UINavigationController, ShareViewDelegate, SAE
             return
         }
 
+        let loadViewController = SAELoadViewController(delegate: self)
+        self.loadViewController = loadViewController
+
+        // Don't display load screen immediately, in hopes that we can avoid it altogether.
+        after(seconds: 2).then { () -> Void in
```

Comment:
2 seconds feels too long for our SAE to apparently do nothing/not appear.  2 seconds is long enough for users to try to launch the SAE again (or another share extension). I'd suggest .4 or .5 seconds.

Reply:
sure, done.